### ISSUE TITLE
Deploy the fix for noisy graphs/holding pen to those installs that are producing output

### ISSUE THREAD

**Issue by crowbot** [mysociety/alaveteli#3299]
- [x] Turn off graphs for Nicaragua
- [x] Deploy fix to Italy
- [x] Deploy fix to Colombia

**Comment by lizconlan** [comment]
* hotfix/0.24.1.5 merged to master
* master merged into develop
* master merged into wdtk

not deployed anywhere yet

**Comment by lizconlan** [comment]
deploying this to Italy dependent on https://github.com/mysociety/dirittodisapere-theme/issues/92
colombia currently on 0.24.0.0

**Comment by lizconlan** [comment]
Deployed to Colombia, now just waiting on the Italian site upgrade mysociety/dirittodisapere-theme#92

**Comment by lizconlan** [comment]
Can be closed now the Italian site has been upgraded
